packages/esm-sms-app/src/index.ts

```diff
@@ -0,0 +1,39 @@
+import { defineConfigSchema, getAsyncLifecycle } from '@openmrs/esm-framework';
+import { configSchema } from './config-schema';
+
+const moduleName = '@openmrs/esm-sms-app';
+
+const options = {
+  featureName: 'provider-settings',
+  moduleName,
+};
+
+export const importTranslation = require.context('../translations', true, /.json$/, 'lazy');
+
+export const providersDashboard = getAsyncLifecycle(() => import('./providers/providers-dashboard.component'), options);
+
+export const smsAdminCardLink = getAsyncLifecycle(() => import('./sms-admin-card-link.component'), options);
+
+export const testProviderConfigForm = getAsyncLifecycle(
+  () => import('./providers/test-form/provider-config-test-form.workspace'),
+  options,
+);
+
+export const addProviderConfigForm = getAsyncLifecycle(
+  () => import('./providers/add-config-form/provider-config-form.component'),
```

Comment:
This file should be `provider-config-form.workspace.tsx`, not `provider-config-form.component.tsx`